CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ After this we will switch probably to real [Semantic Versioning 2.0.0](http://se
 
 ###3.5.38
 * Feature 1209: Added flag fabric8.openshift.generateRoute which if set to false will not generate route.yml and also will not add Route resource in openshift.yml. If set to true or not set, it will generate rou  te.yml and also add Route resource in openshift.yml. By default its value is true.
+* Feature 1206: Added support for spring-boot 2 health endpoint
```

Comment:
Shouldn't this go up into a new section for a new release ?

Reply:
I'm glad it can help :)

Yes, 5.3.38 has been released..